SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -567,22 +557,22 @@ void CNEMOEulerSolver::Upwind_Residual(CGeometry *geometry, CSolver **solver_con
           Project_Grad_j -= Vector_ij[iDim]*Gradient_j[iVar][iDim];
         }
 
-        if (limiter) {
-          if (van_albada) {
-            su2double V_ij = V_j[iVar] - V_i[iVar];
-            Limiter_i[iVar] = V_ij*( 2.0*Project_Grad_i + V_ij) / (4*pow(Project_Grad_i, 2) + pow(V_ij, 2) + EPS);
-            Limiter_j[iVar] = V_ij*(-2.0*Project_Grad_j + V_ij) / (4*pow(Project_Grad_j, 2) + pow(V_ij, 2) + EPS);
-          }
-          if (lim_i > Limiter_i[iVar] && Limiter_i[iVar] != 0) lim_i = Limiter_i[iVar];
-          if (lim_j > Limiter_j[iVar] && Limiter_j[iVar] != 0) lim_j = Limiter_j[iVar];
```

Comment:
This was forcing 1st order because lim_i and lim_j were init to 0

Reply:
@fmpmorgado you were working on this before, maybe you can double-check if this is ok?

Reply:
Thank you for pointing this out @pcarruscag . This has been corrected on feature_NEMO but looks like the fix was not pushed to develop.

Also, @WallyMaier following your last year commit on this part that I just noticed now. As it is, we are using a more conservative approach in feature_NEMO than in develop. In feature_NEMO we are using the same limiter for all the primitive variables (the limiter with lowest value), while in develop we are using a different limiter for each primitive.

For now, I don't see any issues, but we may want to keep this in mind.

Reply:
Ok, I can update the residuals of the two cases failing then? They should also work with 2nd order? Otherwise I can switch the config to 1st order but that is not ideal for coverage.

Reply:
@fmpmorgado feature_NEMO is very out-of-date with develop, so It may be a good idea to update that to stay consistent. 

@pcarruscag, the residuals can be updated, we should be able to do 2nd order.  Thanks for finding this bug!